extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/operations/KuduWriter.java

```diff
@@ -29,6 +24,7 @@ public KuduWriter(String tableName, KuduOperations operations) {
 
   @Override
   public synchronized void write(GeoWaveRow[] rows) {
+    setAutoFlushMode();
```

Comment:
We're currently getting `java.lang.IllegalArgumentException: Cannot change flush mode when writes are buffered`. It looks like we need to only change the flush mode once before we write rows for the first time, otherwise we'll run into this issue.